### PR TITLE
Add semgrep cli help text for SEO

### PR DESCRIPTION
**Pull request by brendongo**

### CODE REVIEW

**Review by DrewDennison — APPROVED** [review]

File: docs/help-text.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# Other arguments
```
Comment (pabloest) [inline docs/help-text.md:1]: How about naming this title and the file something like “CLI reference”?

**Review by pabloest — COMMENTED** [review]
Love this idea, @brendongo! Thank you. I left two comments.

File: docs/help-text.md (hunk 1 above)
Comment (brendongo) [reply docs/help-text.md]: Changed!

**Comment by brendongo** [comment]
Made suggested changes

**Comment by pabloest** [comment]
Thanks, @brendongo !

**Comment by pabloest** [comment]
@brendongo I just realized we already have this page: https://semgrep.dev/docs/cli-usage/

How about merging the content? I would propose moving that content (exit codes, primarily) into the file you created (`cli-reference`).

**Pull request merged**